sirepo/auth_db.py

```diff
@@ -252,6 +274,13 @@ def init_model(callback):
         UserDbBase.metadata.create_all(_engine)
 
 
+@contextlib.contextmanager
+def session_context():
```

Comment:
This should just be called session(), not session_context(). The implementation is a detail to the clients of session(). They need to call it in a with, but that will raise an exception when it's used outside a with.